inspirehep/manage.py

```diff
@@ -64,7 +65,19 @@ def reindexhp():
 def reindex():
     """Reindex all records."""
     from invenio_records.models import RecordMetadata
-    from invenio_records.tasks.index import index_record
+    from invenio_records.tasks.index import get_record_index
+    from invenio_ext.es import es
+    from invenio_records.signals import before_record_index
+    from invenio_base.globals import cfg
```

Comment:
Import inside of functions hinder testability, since they are harder to mock. I'd say that we should do this only when we would get circular imports (but that should be a reason to refactor).

Can you try putting as many of these in the outermost scope as possible?